packages/epics/src/governance/components/document-details.tsx

```
@@ -68,7 +68,7 @@ export const DocumentDetails = ({
         className="rounded-lg"
       >
         <div className="text-2 text-neutral-11">
-          <MarkdownSuspense content={description || ''} />
+          <MarkdownSuspense children={description || ''} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)